Number_types/test/Number_types/eigen.cpp

```diff
@@ -59,7 +59,7 @@ int main(){
   //check<CGAL::Lazy_exact_nt<CGAL::Quotient<CGAL::MP_Float> > >();
   check<CGAL::Sqrt_extension<double,double> >();
 #ifdef CGAL_USE_GMP
-  check<CGAL::Gmpz>();
+//  check<CGAL::Gmpz>();
```

Comment:
???

Reply:
Eigen does not work on integer types. As this file was never tested before, this never raised an error but it does not work.

I could have removed it completely but I figured I'd just comment it (similarly to line 59) as someone might think the test is just missing and no deactivated voluntarily.